test/snippets/expected/Windows Virtual Machine.snippetresult.json

```diff
@@ -1,182 +1,201 @@
+					"adminUsername": "[parameters('adminUsername')]",
+					"adminPassword": "[parameters('adminPassword')]"
 				},
 				"storageProfile": {
 					"imageReference": {
 						"publisher": "MicrosoftWindowsServer",
 						"offer": "WindowsServer",
-						"sku": "2012-R2-Datacenter",
+						"sku": "[parameters('OSVersion')]",
 						"version": "latest"
 					},
 					"osDisk": {
-						"name": "windowsVM1OSDisk",
-						"caching": "ReadWrite",
-						"createOption": "FromImage"
-					}
+						"createOption": "FromImage",
+						"managedDisk": {
+							"storageAccountType": "StandardSSD_LRS"
+						}
+					},
+					"dataDisks": [
+						{
+							"diskSizeGB": 1023,
+							"lun": 0,
+							"createOption": "Empty"
+						}
+					]
 				},
 				"networkProfile": {
 					"networkInterfaces": [
 						{
-							"id": "[resourceId('Microsoft.Network/networkInterfaces', 'windowsVM1-NetworkInterface')]"
+							"id": "[resourceId('Microsoft.Network/networkInterfaces', variables('nicName'))]"
 						}
 					]
 				},
 				"diagnosticsProfile": {
 					"bootDiagnostics": {
 						"enabled": true,
-						"storageUri": "[reference(resourceId('Microsoft.Storage/storageAccounts/', toLower('windowsVM1storage'))).primaryEndpoints.blob]"
+						"storageUri": "[reference(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2022-05-01').primaryEndpoints.blob]"
+					}
+				},
+				"securityProfile": "[if(equals(parameters('securityType'), 'TrustedLaunch'), variables('securityProfileJson'), null())]"
+			},
+			"dependsOn": [
+				"[resourceId('Microsoft.Network/networkInterfaces', variables('nicName'))]",
+				"[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName'))]"
+			]
+		},
+		{
+			"condition": "[and(equals(parameters('securityType'), 'TrustedLaunch'), and(equals(variables('securityProfileJson').uefiSettings.secureBootEnabled, true()), equals(variables('securityProfileJson').uefiSettings.vTpmEnabled, true())))]",
+			"type": "Microsoft.Compute/virtualMachines/extensions",
+			"apiVersion": "2022-03-01",
+			"name": "[format('{0}/{1}', parameters('vmName'), variables('extensionName'))]",
+			"location": "[parameters('location')]",
+			"properties": {
+				"publisher": "[variables('extensionPublisher')]",
+				"type": "[variables('extensionName')]",
+				"typeHandlerVersion": "[variables('extensionVersion')]",
+				"autoUpgradeMinorVersion": true,
+				"enableAutomaticUpgrade": true,
+				"settings": {
+					"AttestationConfig": {
+						"MaaSettings": {
+							"maaEndpoint": "[variables('maaEndpoint')]",
+							"maaTenantName": "[variables('maaTenantName')]"
+						}
 					}
 				}
-			}
+			},
+			"dependsOn": [
+				"[resourceId('Microsoft.Compute/virtualMachines', parameters('vmName'))]"
+			]
 		}
 	],
 	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
 	"contentVersion": "1.0.0.0",
 	"variables": {
-		//Insert here: variable
```

Comment:
Are these comments expected to be removed?

Reply:
Doesn't matter, they're testing artifacts and are removed before baseline comparison anyway.